packages/authentication/test/unit/authentication.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from 'testlab';
+import {Authentication} from '../../src/authentication';
+import {BasicStrategy} from 'passport-http';
+const http = require('http');
+
+describe('Authentication', () => {
+  let passport;
```

Comment:
This creates a variable typed as `any`, which defeats the reasons why we choose TypeScript. Please specify the type of all four variables (`passport`, `strategy`, `req`, `res`), e.g.

```ts
let passport: Authentication;
```